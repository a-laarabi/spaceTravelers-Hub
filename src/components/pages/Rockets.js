import Header from '../Header';

const Rockets = () => (
  <div>
    <Header />
  </div>
);
export default Rockets;
