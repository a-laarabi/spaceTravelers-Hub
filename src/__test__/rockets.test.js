import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import Rockets from '../components/pages/Rockets';

describe('Test for Rockets components', () => {
  it('Check if the Rockets section are available', () => {
    const mission = renderer.create(
      <Provider store={store}>
        <Rockets />
      </Provider>,
    );
    expect(mission).toMatchSnapshot();
  });
});
