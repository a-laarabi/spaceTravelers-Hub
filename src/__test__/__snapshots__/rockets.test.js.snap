// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Rockets components Check if the Rockets section are available 1`] = `<main />`;
